app/assets/stylesheets/layout/_toolbar.sass

```diff
@@ -66,7 +67,7 @@
   display: table
 
 .toolbar-items
-  li
+  > li
```

Comment:
This is the change that causes the WP button group to be broken, since there the li is not directly within `.toolbar-items`.